### ISSUE TITLE
Screen reader is announcing the selected dropdown item as unselected.

### ISSUE THREAD

**Issue by aasimkhan30** [microsoft/vscode#152228]
Does this issue occur when all extensions are disabled?: Yes/No

- VS Code Version:  1.68
- OS Version: win32

Steps to Reproduce:

1. Open a project with launch options.
2. Go to run and debug pane
3. Expand the run and debug dropdown.
4. The currently selected dropdown will be read as unselected by NVDA screen reader 

![image](https://user-images.githubusercontent.com/6816294/173883140-1e6832cb-5e45-4135-92b9-c255f8ddfd4a.png)

This is an accessibility issue opened in Azure Data Studio
Original Issue: https://github.com/microsoft/azuredatastudio/issues/19676

**Comment by isidorn** [comment]
@aasimkhan30 thanks for filling this. There is currently a nice PR by @alanrenmsft which we are discussing, so let's see how that goes.

**Comment by gdomingu** [comment]
Just wanted to see if there might be any plans to revisit this issue, since it is also happening for the inline lightbulb menu, specifically the sparkle one that appears with Copilot.

**Comment by gdomingu** [comment]
@isidorn and @meganrogge any updates on this? We have an accessibility violation about this.

https://github.com/github/accessibility-audits/issues/7439

**Comment by meganrogge** [comment]
@rperez030 or @jooyoungseo, is this a big issue for you? I notice when I test a dropdown in the web, I don't hear unselected or selected at all - just the item name.

**Comment by meganrogge** [comment]
We're missing `aria-selected`

**Comment by rperez030** [comment]
> [@rperez030](https://github.com/rperez030) or [@jooyoungseo](https://github.com/jooyoungseo), is this a big issue for you? I notice when I test a dropdown in the web, I don't hear unselected or selected at all - just the item name.

I honestly don't understand what problem we are trying to solve. The only instance where the selected state of an option in a combobox matters is in the case of multiselect comboboxes, which is not the case with the run and debug dialog. I suspect this is an invalid bug. If we want to know how this is handled, add a select tag to a sample page, add a couple of options, and try navigating using a keyboard and screen reader. It is important to clarify that the UX in macOS and Windows is totally different here.

I'd like to review the original accessibility violation that led to this work. We can probably close it as invalid.

**Comment by meganrogge** [comment]
Thank you so much @rperez030. I think the author of this issue was concerned users would be confused by the term "unselected" being announced. But what you said makes sense as this is a single selection dropdown.

**Comment by gdomingu** [comment]
Hi there, just chiming in from GitHub Copilot team. GitHub has a sev 2 accessibility issue similar to this where the `Show Code Actions` menu is announcing "unselected" when navigating over each of the menu items. 
 
Issue here https://github.com/microsoft/vscode/issues/231070.

@rperez030 am I correct in my interpretation that are you suggesting that neither "selected" nor "unselected" should be read for these types of menus? 

At the very least isn't it an accessibility violation to read incorrect information?

Please let me know if my understanding is incorrect about why this bug is valid.

**Comment by rperez030** [comment]
> [@rperez030](https://github.com/rperez030) am I correct in my interpretation that are you suggesting that neither "selected" nor "unselected" should be read for these types of menus?

Yes. Even if this was a multi select control, most screen readers only announce one of the 2 states. Here is an example if you want to test it further: [https://developer.microsoft.com/en-us/fluentui#/controls/web/combobox](https://developer.microsoft.com/en-us/fluentui#/controls/web/combobox)

> 
> At the very least isn't it an accessibility violation to read incorrect information?

Potentially yes, but without real user impact in this case. The best we can say is that probably the markup that causes this behavior is not ideal, but it has no real implication because this is not a situation where selection matters. I definitely wouldn't consider this as a Sev 2.

If we are fixing this, i suggest using the opportunity to remove the message "Enter to Apply". Which is present for every option in the code actions widget. Having these messages for screen reader users is the equivalent of adding a visual label for mouse users to each option that reads "click to apply".

**Comment by gdomingu** [comment]
Thanks for the information @rperez030. I think that clears up some questions I had.

> Potentially yes, but without real user impact in this case. The best we can say is that probably the markup that causes this behavior is not ideal, but it has no real implication because this is not a situation where selection matters.

I see. For the menu in VS Code, how do the users know that it is not a situation where selection matters? Is it clear when the type of element is announced? I know I was confused by the screenreader announcing "selected, not selected". But I know I'm not an experienced user.



On a separate but related topic, when I tested the multi-select ComboBox on that web link on Mac using VoiceOver it doesn't announce if the checkboxes are selected. That might be a separate bug. Worked find for me with NVDA though.

**Comment by meganrogge** [comment]
Closing as this is not an issue for screen reader users
